.github/workflows/cache-test-images.yaml

```
@@ -0,0 +1,80 @@
+name: Cache test images
+on:
+  workflow_dispatch:
+
+env:
+  GO_VERSION: '1.22'
+jobs:
+  test-images:
+    name: Cache test images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4.1.6
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

We use `go-version` for security patches in the release workflow, but I think we can use `go-version-file` for this workflow.

```suggestion
          go-version-file: go.mod
```

---

I also thought about this.
Updated in https://github.com/aquasecurity/trivy/pull/7614/commits/91a8016b42a752ace91dd904b194d56005931bb4